tests/models/markuplm/test_tokenization_markuplm.py

```diff
@@ -2152,7 +2140,9 @@ def test_maximum_encoding_length_single_input(self):
                     truncated_sequence = information["input_ids"][0]
                     overflowing_tokens = information["input_ids"][1]
                     xpath_tags_seq = information["xpath_tags_seq"][0]
+                    xpath_subs_seq = information["xpath_subs_seq"][0]
                     overflowing_xpath_tags_seq = information["xpath_tags_seq"][1]
+                    overflowing_xpath_subs_seq = information["xpath_subs_seq"][1]
```

Comment:
I had removed those 2 variables, since they are defined but not used anywhere.

Running `make quality `will also tell you that.